PowerShellTools/DebugEngine/Engine.cs

```diff
@@ -462,7 +429,6 @@ int IDebugEngineLaunch2.ResumeProcess(IDebugProcess2 process)
         int IDebugEngineLaunch2.TerminateProcess(IDebugProcess2 process)
         {
             Log.Debug("Engine: TerminateProcess");
-            _events.ProgramDestroyed(_node);

```

Comment:
Don't need this anymore as this will be called after remote process finishes and calling back.